ios/Framework/Core/Offline/CacheManager.swift

```diff
@@ -54,31 +54,42 @@ public enum CacheStrategyType: String {
 	}
 
 	public func getString(collection collection: String, key: String, result: String? -> ()) {
-		readConnection.readWithBlock { transaction in
-			let value: AnyObject? = transaction.objectForKey(key, inCollection: collection)
-			result((value as? NSObject)?.description)
-		}
+
+		var value: AnyObject?
+		readConnection.asyncReadWithBlock( { transaction in
+				value = transaction.objectForKey(key, inCollection: collection)
+			}) {
```

Comment:
when you have more than one closure, don't use the trailing closure syntax. It's quite confusing...